src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -1,325 +1,201 @@
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Step monitor
-
-        API name: `STEP_MONITOR`
-      </td>
-
-      <td>
-        Step monitors are advanced monitors which require no code to set up.
-
-        The monitor can be configured to:
-
-        * Assert modal
-        * Assert text
-        * Assert title
-        * Assert an element
-        * Click an element
-        * Dismiss a modal
-        * Double click an element
-        * Enter a secure credential
-        * Hover over an element
-        * Locate an element by CSS class, HTML ID, link text, Xpath, or value
-        * Navigate to a URL
-        * Select from a dropdown
-        * Type text
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Simple browser monitors
-
-        API name: `BROWSER`
-      </td>
-
-      <td>
-        Simple browser monitors are simple, pre-built scripted browser monitors. They make a request to your site using an instance of Google Chrome.
+<figcaption>
+Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** > search for **Synthetic monitoring** > choose **Ping monitor**. 
+</figcaption>
+</Side>
+</SideBySide>
 
-        Compared to a simple ping monitor, this is a more accurate emulation of an actual customer visit. The user agent is identified as `Google Chrome`.
-      </td>
-    </tr>
+</Step>
 
-    <tr>
-      <td>
-        Scripted browser monitors
+<Step>
 
-        API name: `SCRIPT_BROWSER`
-      </td>
-
-      <td>
-        Scripted browser monitors are used for more sophisticated, customized monitoring. You can create a custom script that navigates your website, takes specific actions, and ensures specific resources are present.
-
-        The monitor uses Google Chrome browser. You can also use a variety of [third-party modules](/docs/synthetics/new-relic-synthetics/scripting-monitors/scripted-monitor-runtime-environment#runtime-table) to build your custom monitor.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        API tests
-
-        API name: `SCRIPT_API`
-      </td>
-
-      <td>
-        API tests are used to monitor your API endpoints. This can ensure that your app server works in addition to your website. New Relic uses the [`http-request` module](https://github.com/request/request) internally to make HTTP calls to your endpoint and validate the results.
-      </td>
-    </tr>
-  </tbody>
-</table>
-
-You can [add and edit synthetic monitors](/docs/synthetics/synthetic-monitoring/using-monitors/add-edit-monitors) directly in our UI. You can also create some synthetic monitors with [Terraform](https://developer.newrelic.com/automate-workflows/get-started-terraform/). To learn how, watch this short YouTube video (approx. 3:25 minutes).
+## Check page load performance with simple browser monitors 
```

Comment:
step monitor -- why would we do a ping vs simple, what is the progression here -- check that w brian (simple, step, scripted browser), find a more organic progression